js/src/main/scala/scalatags/jsdom/SvgTags.scala

```diff
@@ -2,82 +2,83 @@ package scalatags
-  val polygon = "polygon".tag[dom.SVGPolygonElement]
-  val polyline = "polyline".tag[dom.SVGPolylineElement]
-  val radialgradient = "radialgradient".tag[dom.SVGRadialGradientElement]
-  val rect = "rect".tag[dom.SVGRectElement]
-  val set = "set".tag[dom.SVGElement]
-  val stop = "stop".tag[dom.SVGStopElement]
-  val svg = "svg".tag[dom.SVGSVGElement]
-  val switch = "switch".tag[dom.SVGSwitchElement]
-  val symbol = "symbol".tag[dom.SVGSymbolElement]
-  val text = "text".tag[dom.SVGTextElement]
-  val textpath = "textpath".tag[dom.SVGTextPathElement]
-  val tref = "tref".tag[dom.SVGElement]
-  val tspan = "tspan".tag[dom.SVGTSpanElement]
-  val use = "use".tag[dom.SVGUseElement]
-  val view = "view".tag[dom.SVGViewElement]
-  val vkern = "vkern".tag[dom.SVGElement]
+  val altglyph = "altglyph".tag[SVG]
+  val altglyphdef = "altglyphdef".tag[SVG]
+  val altglyphitem = "altglyphitem".tag[SVG]
+  val animate = "animate".tag[SVG]
+  val animatemotion = "animatemotion".tag[SVG]
+  val animatetransform = "animatetransform".tag[SVG]
+  val circle = "circle".tag[Circle]
+  val clippath = "clippath".tag[ClipPath]
+  val `color-profile` = "color-profile".tag[SVG]
+  val cursor = "cursor".tag[SVG]
+  val defs = "defs".tag[Defs]
+  val desc = "desc".tag[Desc]
+  val ellipse = "ellipse".tag[Ellipse]
+  val feblend = "feblend".tag[FEBlend]
+  val fecolormatrix = "fecolormatrix".tag[FEColorMatrix]
+  val fecomponenttransfer = "fecomponenttransfer".tag[ComponentTransferFunction]
+  val fecomposite = "fecomposite".tag[FEComposite]
+  val feconvolvematrix = "feconvolvematrix".tag[FEConvolveMatrix]
+  val fediffuselighting = "fediffuselighting".tag[FEDiffuseLighting]
+  val fedisplacementmap = "fedisplacementmap".tag[FEDisplacementMap]
+  val fedistantlighting = "fedistantlighting".tag[FEDistantLight]
+  val feflood = "feflood".tag[FEFlood]
+  val fefunca = "fefunca".tag[FEFuncA]
+  val fefuncb = "fefuncb".tag[FEFuncB]
+  val fefuncg = "fefuncg".tag[FEFuncG]
+  val fefuncr = "fefuncr".tag[FEFuncR]
+  val fegaussianblur = "fegaussianblur".tag[FEGaussianBlur]
+  val feimage = "feimage".tag[FEImage]
+  val femerge = "femerge".tag[FEMerge]
+  val femergenode = "femergenode".tag[FEMergeNode]
+  val femorphology = "femorphology".tag[FEMorphology]
+  val feoffset = "feoffset".tag[FEOffset]
+  val fepointlight = "fepointlight".tag[FEPointLight]
+  val fespecularlighting = "fespecularlighting".tag[FESpecularLighting]
+  val fespotlight = "fespotlight".tag[FESpotLight]
+  val fetile = "fetile".tag[FETile]
+  val feturbulance = "feturbulance".tag[FETurbulence]
+  val filter = "filter".tag[Filter]
+  val font = "font".tag[SVG]
+  val `font-face` = "font-face".tag[SVG]
+  val `font-face-format` = "font-face-format".tag[SVG]
+  val `font-face-name` = "font-face-name".tag[SVG]
+  val `font-face-src` = "font-face-src".tag[SVG]
+  val `font-face-uri` = "font-face-uri".tag[SVG]
+  val foreignobject = "foreignobject".tag[SVG]
+  val g = "g".tag[G]
+  val glyph = "glyph".tag[SVG]
```

Comment:
These should all by `svg.Element` rather than `svg.SVG`; the latter stands for `dom.SVGSVGElement`, or the `<svg>` tag, only